Library/Homebrew/os/mac/pkgconfig/10.14/libxslt.pc

```diff
@@ -0,0 +1,12 @@
+prefix=/usr
+exec_prefix=${prefix}
+libdir=${exec_prefix}/lib
+includedir=${prefix}/include
+
+
+Name: libxslt
+Version: 1.1.29
+Description: XSLT library version 2.
+Requires: libxml-2.0
+Libs: -L${libdir} -lxslt  -lxml2 -lz -lpthread -licucore -lm
```

Comment:
looks like there are two spaces between xslt and xml2

Reply:
Yeah, it's just a direct copy from the prior `pkg-config` set in this case since there's no version change as far as I can see. I can fix it if ya like but I think the double spacing goes back to at least 10.10 and possibly earlier.

Reply:
👀

Reply:
Amusingly, the double spacing seems to be an upstream thing originally, and we've just kept it over the years.